package.json

```diff
@@ -79,6 +79,7 @@
 		"prewpandroid": "rm -Rf $TMPDIR/gbmobile-wpandroidfakernroot && mkdir $TMPDIR/gbmobile-wpandroidfakernroot && ln -s $(cd \"$(dirname \"../../../\")\"; pwd) $TMPDIR/gbmobile-wpandroidfakernroot/android",
 		"wpandroid": "cd gutenberg && react-native run-android --root $TMPDIR/gbmobile-wpandroidfakernroot --variant wasabiDebug --appIdSuffix beta --appFolder WordPress --main-activity=ui.WPLaunchActivity",
 		"test": "cross-env NODE_ENV=test jest --verbose --config ./jest.config.js",
+		"test:update": "cross-env NODE_ENV=test jest --verbose --config ./jest.config.js --updateSnapshot",
```

Comment:
I added this command to allow Jest snapshots to be updated, similar to the command we have in Gutenberg ([reference](https://github.com/WordPress/gutenberg/blob/05ad081521b040c7ba5ff5726dd16af9523b504e/packages/react-native-editor/package.json#LL109C4-L109C11)).